src/astro/__init__.py

```diff
@@ -1,6 +1,6 @@
 """A decorator that allows users to run SQL queries natively in Airflow."""
 
-__version__ = "1.0.0b1"
+__version__ = "1.1.0a1"
```

Comment:
@kaxil it should be 1.0.0, right?

Reply:
No, this is for main branch -- so it should be the next minor version so we can differentiate between what is released and what is next